PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,30 @@
-Type: patch
+### PR Titles
+#### To pass testing pipeline, these need to follow Conventional Commits spec:
+https://www.conventionalcommits.org/en/v1.0.0/
+e.g.
+`feat: create NewForm component.`
+or
+`fix: update broken link in NewForm component.`
```

Comment:
Just spotted one last thing, our commit message style is without a fullstop at the end. Would be good to match this in the examples here.

Reply:
Yeah cool, all done. Thanks Seb.